test/browser/babel.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  // extends
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        useBuiltIns: 'usage',
+        corejs: '3.6',
+      },
+    ],
+    '@babel/preset-typescript',
+    '@babel/preset-react',
+  ],
+  plugins: [
+    '@babel/plugin-transform-runtime',
+    'babel-plugin-styled-components',
+    // 'babel-plugin-istanbul',
+    // [
+    //   '@babel/plugin-transform-modules-commonjs',
+    //   {
+    //     loose: true,
+    //   },
+    // ],
```

Comment:
Still needed for reference?

Reply:
I don't think I need that anymore (I think that was for trying to get code coverage working). I'll remove it 👍